manual/adobeshockwaveplayer/adobeshockwaveplayer.nuspec

```diff
@@ -1,29 +1,31 @@
-<?xml version="1.0"?>
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>adobeshockwaveplayer</id>
-    <version>12.3.4.204</version>
+    <version>12.3.5.205</version>
+    <packageSourceUrl>https://github.com/AndreAugustoAAQ/chocolatey-packages/tree/main/adobeshockwaveplayer</packageSourceUrl>
+    <owners>André Augusto,pauby,dtgm</owners>
     <title>Adobe Shockwave Player (Final Version - EOL)</title>
-    <authors>Adobe</authors>
-    <owners>pauby,dtgm</owners>
-    <licenseUrl>http://www.adobe.com/products/players/shockwave-player-licensing-distribution.html</licenseUrl>
+    <authors>Adobe Systems Incorporated</authors>
     <projectUrl>http://get.adobe.com/shockwave/</projectUrl>
-    <iconUrl>https://cdn.jsdelivr.net/gh/pauby/chocopackages/icons/adobeshockwaveplayer.png</iconUrl>
+	<iconUrl>https://cdn.jsdelivr.net/gh/AndreAugustoAAQ/chocolatey-packages@main/adobeshockwaveplayer/adobeshockwaveplayer.svg</iconUrl>
+    <licenseUrl>http://www.adobe.com/products/players/shockwave-player-licensing-distribution.html</licenseUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <tags>adobe shockwave adobe-director admin</tags>
+    <summary>Displays Web content that has been created by Adobe Director</summary>

```

Comment:
Please remove the tags and summary from your changes as they haven't been changed.

Reply:
<summary> hasn't been changed

Reply:
done

Reply:
Both the tags and summary have been changed. That's why they are green with the plus next to them.